### DISCUSSION TITLE
Strategy for store params between pages

### DISCUSSION THREAD

**Discussion by electroheadfx** [47ng/nuqs#547] [Q&A]
I'd like to store the nuqs params between nextjs pages, for exemple I go from `/overview?bar=1` to `/history` then I return to `/overview` How I could store bar on overview route, so it switch uri back to `/overview?bar=1` if there is a `bar` cookie/local storage param.
I wonder if anyone already tested a strategy on this case ?

Here some test with getServerSideProps:
https://github.com/vercel/next.js/discussions/11281

**Comment by condinoaljoseph** [comment]
Hi I just realize I need to implement this also on our project. Do you have any idea how you implement this?
